src/scene/scene.js

```diff
@@ -694,14 +756,27 @@ class Scene extends EventHandler {
         this.lightmapMaxResolution = render.lightmapMaxResolution;
         this.lightmapMode = render.lightmapMode;
         this.exposure = render.exposure;
+        this.skyType = render.skyType ?? SKYTYPE_INFINITE;
         this._skyboxIntensity = render.skyboxIntensity ?? 1;
         this._skyboxLuminance = render.skyboxLuminance ?? 20000;
         this._skyboxMip = render.skyboxMip ?? 0;
 
+        if (render.skyboxScale) {
```

Comment:
here the sky properties should be handled similarly to lighting properties.
So based on 
`this.lighting.applySettings(render);` it should be
`this.sky..applySettings(render);`

Reply:
Ahh thanks yea I see. Should i move the existing properties over too like rotation and intensity?

Reply:
Not in this PR. It would change the existing public API, and so needs different handling.

Reply:
Alright thanks